tox.ini

```diff
@@ -66,14 +62,14 @@ deps =
 [testenv:build]
 skip_install = true
 commands =
-    # clean up build/dist folders
+# clean up build/dist folders
     python -c 'import shutil; shutil.rmtree("build", ignore_errors=True)'
     python -c 'import shutil; shutil.rmtree("dist", ignore_errors=True)'
-    # make sure we aren't missing some files
+# make sure we aren't missing some files
     check-manifest
-    # build sdist/wheel
+# build sdist/wheel
     python -m pep517.build .
-    # verify all is well
+# verify all is well
```

Comment:
not sure why these are untabbed now, they were better at the same indentation level

Reply:
I told PyCharm to reformat, which was dumb. Fixed.

Reply:
Not actually fixed, but I will do it myself after merging.